packages/compass-components/src/components/workspace-tabs/tab.tsx

```diff
@@ -126,8 +126,8 @@ const tabIconFocusedStyles = css({
 const tabTitleContainerStyles = css({
   marginRight: spacing[1],
   display: 'inline-grid',
-  paddingTop: spacing[2],
-  paddingBottom: spacing[1] + spacing[2],
+  paddingTop: spacing[3],
+  paddingBottom: spacing[3] - 2, // steal space for the border effect
```

Comment:
```suggestion
  paddingTop: spacing[3] - 2, // steal space for the border effect
  paddingBottom: spacing[3],
```

Was looking at it and I found the top padding to outweigh the bottom a bit. How does this look to you?
Before / after:
<img width="607" alt="Screen Shot 2022-09-16 at 10 17 19 AM" src="https://user-images.githubusercontent.com/1791149/190661413-594d778b-6d0e-4533-8b70-df5e9fa56484.png">
<img width="561" alt="Screen Shot 2022-09-16 at 10 19 31 AM" src="https://user-images.githubusercontent.com/1791149/190661447-397eadc0-904b-478b-98fe-04ba62fafd8a.png">